jwcrypto/jwt.py

```diff
@@ -130,10 +167,12 @@ def __init__(self, header=None, claims=None, jwt=None, key=None,
         :param check_claims: An optional dict of claims that must be
          present in the token, if the value is not None the claim must
          match exactly.
+        :param keyset: A (:class:`jwcrypt.jwk.JWTSet`) keyset containing
+         the key to deserialize the token.
```

Comment:
This option should be incompatible with passin a Key, can you ad mutual exclusivity across ?